app/src/main/java/com/concordia/mcga/models/ConnectedPOI.java

```diff
@@ -5,16 +5,28 @@
 
 public class ConnectedPOI {
     private Map<Integer, IndoorPOI> indoorPOIs;
+    private int floorNumber;
```

Comment:
How would I associate a ConnectedPOI with a particular floor, though? For example, the escalator from H4 to H2 is one way (H4 -> H2). It connects 2 points, the escalators as IndoorPOIs, but is located on the 4th floor.

I may have just misunderstood the path finder implementation though, if its not necessary I'll refactor it.